test-file

```diff
@@ -46,4 +46,22 @@ flowchart TD
     B -- Yes --> C[OK]
     C --> D[Rethink]
     D --> B
-    B -- No --> E[End]  // B -- No ----> E[End]
\ No newline at end of file
+    B -- No --> E[End]  // B -- No ----> E[End]
+gantt
```

Comment:
Can `test-file` be moved to an appropriate place? `text/fixtures/gantt1`?

Reply:
Shall I split the file into two? one for ganttchart , one for flowchart, as of now I put them into one and moved the file to `/text/fixtures/`